core/arch/arm/include/kernel/tz_ssvce.h

```diff
@@ -48,10 +48,24 @@ void arm_cl1_i_inv_all(void);
 void arm_cl1_i_inv(void *start, void *end);
 
 void secure_mmu_datatlbinvall(void);
-void secure_mmu_unifiedtlbinvall(void);
-void secure_mmu_unifiedtlbinvbymva(unsigned long addr);
-void secure_mmu_unifiedtlbinv_curasid(void);
-void secure_mmu_unifiedtlbinv_byasid(unsigned long asid);
+
+void tlbi_all(void);
+void tlbi_asid(unsigned long asid);
+void tlbi_mva_allasid(unsigned long addr);
+static inline void tlbi_mva_allasid_nosync(vaddr_t va)
+{
+#ifdef ARM64
+	uint64_t mva = va;
+
+	asm volatile ("tlbi	vaae1is, %0" : : "r" (mva));
```

Comment:
Please add a `write_vaae1is()` to arm64.h and use that function instead. Do the corresponding thing for ARM32 below.